noop/unikernel.ml

```diff
@@ -0,0 +1,6 @@
+module Main = struct
```

Comment:
An other view on the noop unikernel is that we can already go quite far without introducing module/functors at all: we can build "pure" unikernel which just prints stuff using `Logs`, that you can configure using some configure/runtime keys on KVM/Xen/Unix.